rdb/src/main/java/com/instana/dc/rdb/impl/informix/InformixDc.java

```diff
@@ -230,6 +233,7 @@ private void scheduleCustomPollRate(int pollInterval, IntervalType intervalType)
     @SuppressWarnings("unchecked")
     private void parseCustomAttributes(Map<String, Object> properties) {
         Map<String, Object> customInput = (Map<String, Object>) properties.get("custom.input");
+        sqlTraceEnabled = (Boolean)customInput.getOrDefault(DB_SQL_TRACE_ENABLED, false);
```

Comment:
Suggestion: if you are converting it to a wrapper object use `Boolean` data type only for the attribute.